spec/esoteric_formats_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+describe 'Parsing esoteric files and files causing ambiguous detection' do
```

Comment:
Is there any particular reason to do this test here and not add the spec to MP3 parser spec as well? It's not a blocker but I'm curious. (I do like having a specific place for this sort of thing!)

Reply:
Because I believe this is not the last time we discover a file like this. As a matter of fact we can run a contest about crafing a file that would detect as _as many_ file types as possible and all in err.